Controllers/controller.py

```diff
@@ -106,3 +111,20 @@ def open_file_dialog(self):
             url = file_dialog.selectedUrls()[0]
             self._media_player.setSource(url)
             self._media_player.play()
```

Comment:
Can we remove this line? Since we have a play/pause button, the video should no longer automatically play.